### PR TITLE
Add multi-arch docker image support

### PR DESCRIPTION
**Pull request by cyclinder**
support amd64, arm64 and ppc64le platform for sriov-network-operator, sriov-network-operator-config-daemon and sriov-network-operator-webhook.

Fixes https://github.com/k8snetworkplumbingwg/sriov-network-operator/issues/430

### CODE REVIEW

File: .github/workflows/image-push-master.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64,linux/arm64,linux/ppc64le
```
Comment (e0ne) [inline .github/workflows/image-push-master.yml:30]: According to https://github.com/docker/build-push-action/tree/releases/v2?tab=readme-ov-file it's recommended to setup [setup-buildx](https://github.com/docker/setup-buildx-action) action

Comment (cyclinder) [reply .github/workflows/image-push-master.yml]: thanks @e0ne , Updated.

**Review by e0ne — APPROVED** [review]

File: .github/workflows/image-push-master.yml [hunk 2]
```diff
@@ -1,89 +1,128 @@
...
+
       - name: Build and push sriov-network-operator
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v4
         with:
           context: .
           push: true
+          platforms: ${{ env.BUILD_PLATFORMS }}
           tags: |
             ${{ env.IMAGE_NAME }}:latest
```
Comment (adrianchiris) [inline .github/workflows/image-push-master.yml:47]: this wont mess up the tags ?

we want to have a latest tag per platform, then create and push a manifest which will be with "latest" lag containing platform specific images.

**Comment by cyclinder** [comment]
Hi @adrianchiris, Could you please take a look?

**Review by SchSeba — APPROVED** [review]
great work!

**Comment by SchSeba** [comment]
merging this one
@cyclinder do you have some bandwidth to open the same PR for the other sriov repos?
sriov-cni
device-plugin
resource-injector
ib-sriov-cni

Thanks!

**Comment by cyclinder** [comment]
@SchSeba Yes, sure. I can work on this in these days.

**Comment by cyclinder** [comment]
- sriov-cni: https://github.com/k8snetworkplumbingwg/sriov-cni/pull/234
- device-plugin: https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/457
-ib-sriov-cni https://github.com/k8snetworkplumbingwg/network-resources-injector/pull/152

Only resource-injector doesn't have multi-image support, so I opened https://github.com/k8snetworkplumbingwg/ib-sriov-cni/pull/80.

@SchSeba  Would you like to update the CI yaml like these changes?

**Pull request merged**